src/app/App.js

```diff
@@ -1278,13 +1298,10 @@ class App extends EventEmitter {
 	}
 
 	/**
-	 * Cancels pending navigate with <code>Cancel pending navigation</code> error.
+	 * Resets pending navigate.
 	 * @protected
 	 */
 	stopPendingNavigate_() {
-		if (this.pendingNavigate) {
-			this.pendingNavigate.cancel('Cancel pending navigation');
```

Comment:
This probably should not keep? Since the behavior was to issue an error message and "cancel" the promise...